pkgs/top-level/all-packages.nix

```diff
@@ -11157,7 +11157,9 @@ with pkgs;
 
   libxdg_basedir = callPackage ../development/libraries/libxdg-basedir { };
 
-  libxkbcommon = callPackage ../development/libraries/libxkbcommon { };
+  libxkbcommon = libxkbcommon_8;
+  libxkbcommon_8 = callPackage ../development/libraries/libxkbcommon { };
```

Comment:
Introducing `libxkbcommon_8` and aliasing it to `libxkbcommon` shouldn't be done for something as trivial as this.

Reply:
Why that?

Reply:
@Mic92 Seems unnecessary to me, you disagree?

Reply:
It is quite a common pattern in nixpkgs that we keep have aliases like that even if there are only two versions.